### PR TITLE
make install.sh work in windows Git Bash / mingw shell

### PR DESCRIPTION
**Pull request by ccheng**
So I got install.sh to work with Git Bash, which I think is actually mingw shell, and I've tested it on Mac, Windows, and Ubuntu. #68

### CODE REVIEW

**Comment by ccheng** [comment]
Not sure why ci / test (ubuntu-latest) failed.  Got to:
```
Deno was installed successfully to /home/runner/.deno/bin/deno
```
But then
```
./install_test.sh: 17: ./install_test.sh: ./install.sh: Permission denied
##[error]Process completed with exit code 126.
```

File: install.sh [hunk 1]
```diff
@@ -4,9 +4,11 @@
 
 set -e
 
+win_target="x86_64-pc-windows-msvc"
 case $(uname -s) in
 Darwin) target="x86_64-apple-darwin" ;;
-*) target="x86_64-unknown-linux-gnu" ;;
+Linux) target="x86_64-unknown-linux-gnu" ;;
+*) target=$win_target ;;
```
Comment (MarkTiedemann) [inline install.sh:11]: I don't think the default target should be Windows. Wikipedia has a long list of `uname -s` output: https://en.wikipedia.org/wiki/Uname For Linux, there's `FreeBSD`, `GNU`, `DragonFly`. For Windows, Wine uses `Windows_NT`, Cygwin uses `CYGWIN_NT-$version`, MingW uses `MINGW$arch_NT-$version`, MSYS uses `MSYS_NT-$version`.

Maybe `$OS = "Windows_NT"` would be a good check as well? (see: https://en.wikipedia.org/wiki/Environment_variable#Windows)

File: install.sh [hunk 2]
```diff
@@ -20,15 +22,22 @@ if [ $# -eq 0 ]; then
 			command grep -o "/denoland/deno/releases/download/.*/deno-${target}\\.zip" |
 			command head -n 1
 	)
-	if [ ! "$deno_asset_path" ]; then exit 1; fi
+	if [ ! "$deno_asset_path" ]; then
+		echo "Error: deno_asset_path not found."
```
Comment (MarkTiedemann) [inline install.sh:26]:
```suggestion
		echo "Error: Unable to find latest GitHub release."
```

File: install.sh (hunk 1 above)
Comment (ccheng) [reply install.sh:11]: Would it be okay to check for `$OS = "Windows_NT"` then if not Darwin, then default to Linux?

**Pull request closed without merging**